### PR TITLE
SCMOD-15150: Updated release-notes

### PR DESCRIPTION
**Pull request by kyra-ohare**
https://portal.digitalsafe.net/browse/SCMOD-15150

### CODE REVIEW

**Review by michael-bryson — CHANGES_REQUESTED** [review]
Looks like the notes weren't updated for this change: https://github.com/WorkerFramework/worker-framework/commit/657881cf5492d914d585aba512e7f5ce5b0426c5

Sounds like a good feature to mention.

File: release-notes-5.1.0.md
Comment (michael-bryson) [inline release-notes-5.1.0.md]:
```suggestion
- SCMOD-5524: Eliminated base64 encoding of taskData to reduce message size.
```

**Review by michael-bryson — APPROVED** [review]

**Comment by kyra-ohare** [comment]
@michael-bryson
Should I wait for Xavier to merge his [PR](https://github.com/WorkerFramework/worker-framework/pull/127) first before I merge this one? I have asked him about his PR and he said it's still ongoing.
Also, I noticed the copyright in the website says "2015-2018 MICRO FOCUS OR ONE OF ITS AFFILIATES" in the footer. Should I fix it first to "2015-2021 MICRO FOCUS OR ONE OF ITS AFFILIATES"?

**Comment by michael-bryson** [comment]
> @michael-bryson Should I wait for Xavier to merge his [PR](https://github.com/WorkerFramework/worker-framework/pull/127) first before I merge this one? I have asked him about his PR and he said it's still ongoing.

No you don't need to wait to merge the release note change, it is Xaviers responsibility to update the notes for that change. It might block the release but I don't see it listed in the JIRA as a blocking ticket so its probably ok to go ahead. If you want to make sure you can check with Dermot.

**Pull request merged**
